### PR TITLE
[DS-1471] Fix error during creating nodes and adding blocks to LB

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [DS-1471](https://yusa.atlassian.net/browse/DS-1471)

## Steps for review

- [ ] Log in as an admin to the CMS.
- [ ]  Accept the Terms and Conditions if prompted.
- [ ] Attempt to create various content types.
- [ ] Try adding content blocks inside layout builder blocks.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ycloudyusa/yusaopeny/tree/main/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with an account on drupal.org, otherwise, you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Review by rollki — APPROVED** [review]
LGTM

File: composer.json [hunk 1]
```diff
@@ -337,6 +337,7 @@
                 "Issue #3335841 - Admin toolbar menu z-index increase": "https://www.drupal.org/files/issues/2023-02-06/3335841-2.patch"
             },
             "drupal/ckeditor5_font": {
+                "Issue #3368736: no config schema' Error after update to Drupal 10.1.0": "https://www.drupal.org/files/issues/2024-01-11/ckeditor5_font-schema-fix-10.2.1-3368736-37.patch",
```
Comment (podarok) [inline composer.json:340]: Let's always respond in drupal.org issue queue the patch fixed issue for us with setting it to RTBC, or adding comment +1 RTBC if it is already set
/cc @rollki @aleevas

Comment (podarok) [reply composer.json:340]: https://www.drupal.org/node/3368736

**Review by podarok — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rollki) [reply composer.json:340]: @podarok 
Isn't it better to leave a comment after testing? Is it enough that the developer checked it?

Comment (podarok) [reply composer.json:340]: Developer tested on local machine? @rollki 
It is enough

**Comment by podarok** [comment]
Still no comment on drupal.org @rollki @aleevas - please do as a follow-up

We can't stay just takers in community, we need to provide feedback and ensure patches committed and released with our help

**Pull request merged**